src/nidigital/metadata/functions_addon.py

```diff
@@ -4,4 +4,61 @@
 functions_override_metadata = {
 }
```

Comment:
Is this still needed?

Reply:
Right now, no. But I like to leave it there empty for when we do need to add something.